### PR TITLE
Refs 2190: adding enabled check to accessible

### PR DESCRIPTION
**Pull request by xbhouse**
## Summary

Adding check so that if a feature is not enabled, it is also not accessible

## Checklist

- [x] Tested with snapshotting feature disabled and pulp server URL not configured if appropriate

### CODE REVIEW

File: pkg/handler/features.go [hunk 1]
```diff
@@ -57,6 +57,9 @@ func accessible(ctx context.Context, feature config.Feature) bool {
 	if feature.Accounts == nil && feature.Users == nil {
 		return true
 	}
+	if !feature.Enabled {
+		return false
+	}
```
Comment (jlsherrill) [inline pkg/handler/features.go:62]: i'd actually put this above the previous if statement

**Comment by jlsherrill** [comment]
and you'll want to adjust the title to  "Refs 2190"  refs = references, but doesn't fix it.  That way it'll be associated to the jira task, but won't close it.

**Review by jlsherrill — APPROVED** [review]
ACK!

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-2190

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

**Comment by swadeley** [comment]
Hello

see test results here:
[snapshot=True](https://github.com/content-services/content-sources-frontend/pull/155#issuecomment-1791712258)

**Pull request merged**